### PR TITLE
[New][Author] Add stickybits w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by yowainwright**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): [stickybits](https://github.com/dollarshaveclub/stickybits)
 * Official website (optional, not the repository): https://dollarshaveclub.github.io/stickybits/
 * NPM package url (optional): https://www.npmjs.com/package/stickybits
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): https://github.com/dollarshaveclub/stickybits
   - Count of watchers: 31 
   - Count of stars: 1087
   - Count of forks: 63
 * NPM download stats (optional): 
   - Downloads in the last day: 108 
   - Downloads in the last week: 1353
   - Downloads in the last month: 9173

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/stickybits/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "author": "Jeff Wainwright <jjwainwright2@gmail.com> (https://jeffry.in)",
+  "license": "MIT",
+  "homepage": "https://dollarshaveclub.github.io/stickybits/",
+  "npmName": "stickybits",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```
Comment (sufuf3) [inline ajax/libs/stickybits/package.json]: Would you mind telling us that can `stickybits.es.js` be used in web front-end?
If developers can use this file in front-end, then this pattern is good, else we have to exclude it by using `**/!(*.es.*)`.

Thank you.

**Comment by yowainwright** [comment]
@sufuf3 Thanks for taking the time review this PR. 

I'm sorry I have a few more questions (Yes, I've tried to understand the documentation and wiki). 

What is meant by **'I found that this PR is from unknown repository'**?

How would I update the commit message without having to pull the CDNJS repository down? Could I just submit another PR at that point? 

~Thank you very much!

File: ajax/libs/stickybits/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "stickybits",
+  "filename": "stickybits.js",
```
[the remark this answers is not in the document]
Comment (DanielRuf) [reply ajax/libs/stickybits/package.json]: would be helpfulf if you have some Git hook or CI service which checks this ;-)

[the remark this answers is not in the document]
Comment (sufuf3) [reply ajax/libs/stickybits/package.json]: Thank you. Using Git Hooks is a great idea. 😃

**Pull request closed without merging**